course/layouts/resources/list.html

```diff
@@ -0,0 +1,61 @@
+{{ define "main" }}
+{{- $numberOfResourcesLimit := 10 -}}
+{{ partial "course_content.html" . }}
+
+<div class="mb-5">
+  {{ range  .Pages.GroupByParam "resourcetype"  }}
```

Comment:
**Note:** Currently, to get the UI and other functionalities sorted, resources have been groupedBy `resourceType` due to [this](https://github.com/mitodl/hq/discussions/92#discussioncomment-2954821). Once this issue is sorted, then we will group resources by `learning_resource_types`